helm/dagster/schema/schema_tests/test_instance.py

```diff
@@ -984,6 +984,14 @@ def test_retention(template: HelmTemplate):
                     started=4,
                 ),
             ),
+            backfill=TickRetention.construct(
+                purgeAfterDays=TickRetentionByType(
```

Comment:
i copied the retention policies from the other kinds of ticks, but @gibsondan had a good point that we probably want to delete all of the ticks for a backfill in one go. so this might need to change to be like "delete all ticks for completed backfills N days after the backfill completes"

Reply:
The direction we're going with the tick table is to partition it by date.  The retention policy will also be by date, where we'll just drop the old partitions.

This means that if a backfill spans multiple days, when we drop the oldest day partition, it will have only partially deleted the backfill.  This doesn't preclude us from deleting backfill ticks sooner than that, but we wouldn't get those operations for free (we'd have to write a different tick deletion daemon for backfills that would go through and delete them).